device/sycl/src/clusterization/clusterization_algorithm.sycl

```diff
@@ -137,32 +137,33 @@ clusterization_algorithm::output_type clusterization_algorithm::operator()(
         .wait_and_throw();
 
     // Copy number of measurements to host
-    unsigned int num_measurements_host;
+    vecmem::unique_alloc_ptr<unsigned int> num_measurements_host =
+        vecmem::make_unique_alloc<unsigned int>(*(m_mr.host));
```

Comment:
Same comment here. And in a few more places, so I won't point out all of them...